adafruit_bluefruit_connect/packet.py

```diff
@@ -12,7 +12,15 @@
 
 """
 
+from __future__ import annotations
+
 import struct
+from io import RawIOBase
```

Comment:
This import needs to be inside of the try block underneath the typing import. The microcontrollers don't have this module so it will raise an exception (and even if they did we don't want to import it if not needed at runtime so we can save RAM.)